packages/at_auth/lib/src/auth/at_auth_request.dart

```diff
@@ -1,16 +1,62 @@
 import 'package:at_auth/src/keys/at_auth_keys.dart';
+import 'package:at_chops/at_chops.dart';
 import 'package:at_commons/at_commons.dart';
+import 'package:at_auth/src/auth_constants.dart' as auth_constants;
 
 class AtAuthRequest {
   String atSign;
+
   AtAuthRequest(this.atSign);
+
   String? enrollmentId;
   AtAuthKeys? atAuthKeys;
   String rootDomain = 'root.atsign.org';
   int rootPort = 64;
   PkamAuthMode authMode = PkamAuthMode.keysFile;
   String? atKeysFilePath;
+  Map<String, dynamic>? encryptedKeysMap;
 
   /// public key id from secure element if [authMode] is [PkamAuthMode.sim]
   String? publicKeyId;
+
+  AtAuthKeys getDecryptedKeys() {
```

Comment:
please remove this method in order to keep the class simple. we can move this logic elsewhere

Reply:
Removed the method.